### PR TITLE
Add head support

### PR DESCRIPTION
**Pull request by arempter**
Adds support for aws commands using HEAD requests like:
- aws s3api --endpoint $endpoint head-object ...
- updated readme with correct default AWS region
- small refactor of extractRequestParameters method

### CODE REVIEW

File: src/it/resources/application.conf [hunk 1]
```diff
@@ -6,3 +6,11 @@ airlock {
         }
     }
 }
+
+akka {
+    http {
+        # we need to disable head to get translation
```
[the remark this answers is not in the document]
Comment (arempter) [reply src/it/resources/application.conf]: this should be set in both it:test application conf and in main application.conf

we can skip adding it it:test, but then all tests will be GET, if we add it, some tests like listObjectsV2 will be handled as HEAD

[the remark this answers is not in the document]
Comment (nielsdenissen) [reply src/it/resources/application.conf]: ah ok, but if it's in the main application.conf we don't have to also set it in the it:test one right? The main one is always used

File: src/it/scala/com/ing/wbaa/airlock/proxy/handler/RequestHandlerS3ItTest.scala [hunk 2]
```diff
@@ -105,6 +105,19 @@ class RequestHandlerS3ItTest extends AsyncWordSpec with DiagrammedAssertions wit
...
+          withBucket(sdk) { testBucket =>
+            withFile(1024 * 1024) { filename =>
+              val testKeyFile = "keyPutFileByFile"
+
+              sdk.putObject(testBucket, testKeyFile, new File(filename))
+
+              assert(sdk.doesObjectExist(testBucket, testKeyFile))
+            }
+          }
+        }
```
[the remark this answers is not in the document]
Comment (arempter) [reply src/it/scala/com/ing/wbaa/airlock/proxy/handler/RequestHandlerS3ItTest.scala]: removed indeed this test. copyObjects, after small update still fails with:

 com.amazonaws.services.s3.model.AmazonS3Exception: Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: null; S3 Extended Request ID: null)

ranger policy for new bucket! :(

File: src/it/resources/application.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (arempter) [reply src/it/resources/application.conf]: for some reason I had to...

**Pull request merged**